src/Microsoft.Identity.Web.TokenAcquisition/ServiceCollectionExtensions.cs

```diff
@@ -41,6 +41,10 @@ public static class ServiceCollectionExtensions
         {
             _ = Throws.IfNull(services);
 
+#if !NETSTANDARD2_0 && !NET462 && !NET472
```

Comment:
Are you sure that this is correct? Later below it says that #else is a netfwk.
Netstadrdad2_0 can end up being consumed in any NET Core runtime including 7.0 and i imagine the unreleased 8.0

Reply:
Fixed. Thanks!